site/src/pages/setup.js

```diff
@@ -0,0 +1,101 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const SetupPage = ({ data, location }) => {
+  const siteTitle = data.site.siteMetadata.title
+
+  return (
```

Comment:
We need to talk about how I'm supposed to do this properly as this clearly isn't the way, and I haven't had the time to research it.

Reply:
Do which part properly? I've been underwater with other work. Thanks for taking time to do any of this at all. Its immensely helpful.

Reply:
I'm not sure the whole page in html in one chunk is the 'react / gatsby' approach :) 

Yeah no problem. I hope to have some time soon to actually learn a bit of this paradigm, just want to get the content in so we arn't in a rush for the July event.